### PR TITLE
feat(charm): Add completion

### PR DESCRIPTION
**Pull request by mamercad**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New completion for the [Charm](https://charm.sh/) CLI.

**What is the current behavior? (You can also link to an open issue here)**

There is no completion for the Charm CLI.

**What is the new behavior (if this is a feature change)?**

There will be completion for the Charm CLI.

**Additional info:**

### CODE REVIEW

File: src/charm.ts [hunk 1]
```diff
@@ -0,0 +1,273 @@
+const completionSpec: Fig.Spec = {
+  name: "charm",
+  description: "",
```
Comment (SeparateRecords) [inline src/charm.ts:3]: Can you add a description here?

**Review by SeparateRecords — APPROVED** [review]
Awesome, thanks @mamercad! One quick fix then it's good to merge.

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
damnit wrong button

**Review by SeparateRecords — APPROVED** [review]
Beautiful, thank you so much and great job as always @mamercad 😄

**Comment by mamercad** [comment]
> Beautiful, thank you so much and great job as always @mamercad 😄

Hehe, no worries, thanks for the merge!

**Pull request merged**